backend/oasst_backend/config.py

```diff
@@ -77,7 +77,7 @@ class Settings(BaseSettings):
     DEBUG_USE_SEED_DATA_PATH: Optional[FilePath] = (
         Path(__file__).parent.parent / "test_data/realistic/realistic_seed_data.json"
     )
-    DEBUG_ALLOW_SELF_LABELING: bool = False  # allow users to label their own messages
```

Comment:
Thanks for the replies typo fix, but do you think self-labeling is really self-explanatory and it is better to remove the comment?